src/main/java/dao/Impl/DeveloperDaoImpl.java

```diff
@@ -0,0 +1,114 @@
+package dao.Impl;
+
+import model.Company;
+import model.Project;
+import util.ConnectionFactory;
+import dao.DeveloperDao;
+import model.Developer;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashSet;
+import java.util.Set;
+
+public class DeveloperDaoImpl implements DeveloperDao {
+
+    @Override
+    public Developer findOne(Long id) {
+        try (Connection connection = ConnectionFactory.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM user WHERE id=" + id);
+            if (resultSet.next()) {
+                return extractDeveloperFromResultSet(resultSet);
+            }
+        } catch (SQLException e) {
+            System.out.println("Something wrong with util");
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Override
+    public Set findAll() {
+        try (Connection connection = ConnectionFactory.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM developer");
+
+            Set developers = new HashSet();
```

Comment:
I would recommed to use generics in collections
```suggestion
            Set<Developer> developers = new HashSet<>();
```